### ISSUE TITLE
swagger online verification failed

### ISSUE THREAD

**Issue by caryhgq** [fastify/fastify-swagger#321]
## 🐛 Bug Report

Routing schemas are generated using [fluent-schema](https://github.com/fastify/fluent-schema), and documents produced by fastify-Swagger cannot be validated online with Swagger.

## To Reproduce

```js
// schemas file
const { docs } = require("../utils/dict");
const S = require("fluent-schema");

const bodyJsonSchema = S.object()
  .prop("age", S.number().description("年龄"))
  .prop("userName", S.string().description("用户名称"));
 
const signBody = S.object().prop("url", S.string().required().description("分享页面链接"));

const registerSchema = {
  summary: "用户注册接口",
  description: "测试数据库写入操作",
  tags: [docs.example],
  body: bodyJsonSchema,
};

const querySignSchema = {
  summary: "JSSDK签名",
  description: "获取js-sdk签名",
  tags: [docs.wxAuth],
  body: signBody,
};

module.exports = {
  registerSchema,
  querySignSchema 
};

-----------------------
//  route file
const controller = require('../controllers/example');
const schemas = require('../schemas/index');

module.exports = function (fastify, opts, done) {
  fastify.post('/register', { schema: schemas.registerSchema }, controller.register);
  fastify.post("/jsSign",{ schema: querySignSchema }, controller.jsSdkSign);
  done();
}

```
Return after [swagger ]( https://validator.swagger.io/validator/debug) verification (Documents are automatically validated when they are online)

![image](https://user-images.githubusercontent.com/6456736/98914562-caa6f100-2503-11eb-8158-708f34b74ee1.png)

```js
{
    "messages": [
        "attribute paths.'/example/register'(post).[body].$schema is unexpected",
        "attribute paths.'/wx/jsSign'(post).[body].$schema is unexpected"
    ],
    "schemaValidationMessages": [
        {
            "level": "error",
            "domain": "validation",
            "keyword": "oneOf",
            "message": "instance failed to match exactly one schema (matched 0 out of 2)",
            "schema": {
                "loadingURI": "http://swagger.io/v2/schema.json#",
                "pointer": "/definitions/parametersList/items"
            },
            "instance": {
                "pointer": "/paths/~1example~1register/post/parameters/0"
            }
        }
    ]
}
```
swagger output json content

![image](https://user-images.githubusercontent.com/6456736/98913871-ee1d6c00-2502-11eb-86d7-29312baf8cfb.png)
![image](https://user-images.githubusercontent.com/6456736/98913972-0a210d80-2503-11eb-8810-d0df23d6455a.png)

## Expected behavior
Through swagger online verification!

## Your Environment

- *node version*: v14.15.0
- *fastify version*: >=3.8.0
- *fluent-schema version*: >=1.0.4
- *fastify-swagger version*: >= 3.5.0
- *os*: Linux

**Comment by mcollina** [comment]
Interesting, sounds like a bug! 
Would you like to send a Pull Request to address this issue? Remember to add unit tests.

**Comment by climba03003** [comment]
It is not a bug, ```swagger 2.0``` do not support ```oneOf``` and ```anyOf``` schema and this plugin only support ```swagger 2.0```.
#265 need to be done before it can be resolved.

**Comment by mcollina** [comment]
Duplicate of #265
